app/views/objects/show/_attributes.html.slim

```diff
@@ -0,0 +1,20 @@
+div class="md:px-3 my-3"
+  h3 class="lg:text-2xl text-lg text-gray-900 dark:text-gray-200 font-semibold mt-8"
+    = title
+  hr
+- attributes.each do |a|
+  div class="md:p-3 py-3 my-3"
+      a style="display: block; position: relative; top: -80px; visibility: hidden;" id="#{a.name}"
+      div class="flex flex-wrap"
+        div class="w-full md:w-10/12"
+          h4 class="lg:text-2xl text-lg text-gray-900 dark:text-gray-200 font-semibold"
+            a href="##{a.name}"= a.name
+        div class="flex md:justify-end w-full md:w-2/12 mt-3 md:mt-0 font-mono"
+          span class="px-2 h-6 inline-block rounded bg-gray-200 dark:bg-gray-700 algin-middle cursor-default" title="#{a.access}"
+            = a.access.titleize
+      - if a.description.empty?
+        div class="ruby-documentation py-1"
+          | No documentation available
+      - else
+        div class="ruby-documentation py-1"
+          == a.description
```

Comment:
You guys recently discussed ending files with new lines. This should have a newline, right?